common/src/main/java/dev/aurelium/auraskills/common/source/SourceLoader.java

```diff
@@ -194,6 +201,20 @@ public Map<SourceTag, List<XpSource>> loadTags(Skill skill, File contentDirector
         }
     }
 
+    private Collection<XpSource> getFilteredSource(List<XpSource> sources, String rawFilter) {
+        String filter = rawFilter.replace("!", "").replace("*", "").toLowerCase();
```

Comment:
Pass Locale.ROOT into toLowerCase, the method without params is locale sensitive.